biofefi/components/configuration.py

```diff
@@ -345,7 +350,7 @@ def execution_options_box_manual():
     )
     st.write(
         """
-        If you select **"Standardization"**, your data will be normalised by subtracting the
+        If you select **"Standardization"**, your data will be normalized by subtracting the
```

Comment:
```suggestion
        If you select **"Standardization"**, your data will be normalised by subtracting the
```
The code bit is spelled wrong